android/engine/src/test/java/org/smartregister/fhircore/engine/sync/SyncListenerManagerTest.kt

```diff
@@ -62,6 +59,8 @@ class SyncListenerManagerTest : RobolectricTest() {
         configService = configService,
         sharedPreferencesHelper = sharedPreferencesHelper,
         configurationRegistry = configurationRegistry,
+        context = ApplicationProvider.getApplicationContext(),
+        dispatcherProvider = co,
```

Comment:
@ellykits the `dispatcherProvider` value `co` is an unresolved variable, Could you please guide the correct value here?